bootstrap/service_test.go

```diff
@@ -58,6 +58,7 @@ func newService(auth mainflux.AuthServiceClient, url string) bootstrap.Service {
 	things := mocks.NewConfigsRepository()
 	config := mfsdk.Config{
 		BootstrapURL: url,
+		ThingsURL:    url,
```

Comment:
Same question here.

Reply:
answer is similar as above, additionaly providing `ThingsURL` here is configuring bootstrap service so that it can reach `things` service API endpoints to create things, channels, check connections